.travis.scripts/compile.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export CFLAGS="-Wall -Wextra -Wdeclaration-after-statement -Wmissing-field-initializers -Wshadow -Wno-unused-parameter -ggdb3"
+export CFLAGS="-Wall -Wextra -Wdeclaration-after-statement -Wmissing-field-initializers -Wshadow -Wno-unused-parameter -ggdb3 -DDEBUGGER_FAIL_SILENTLY"
```

Comment:
Ugly, but I guess that works. I will see what it does locally before "approving" this though :-)